waveform_editor/gui/selector/options_button_row.py

```diff
@@ -70,6 +70,21 @@ def __init__(
             visible=has_waveforms,
         )
 
+        # 'Rename waveform' button
+        self.rename_waveform_button = pn.widgets.ButtonIcon(
+            icon="cursor-text",
+            size="20px",
+            active_icon="check",
+            description="Rename waveform",
+            on_click=self._on_rename_waveform_button_click,
+            visible=has_waveforms,
+        )
+        self.rename_waveform_panel = TextInputForm(
```

Comment:
This is a bit confusing when changing the waveform selection after pressing the rename button:

https://github.com/user-attachments/assets/883c8c31-4282-453a-9412-c9294074f587

Could we do this in a modal instead?